libs/barista-components/progress-bar/src/progress-bar.spec.ts

```diff
@@ -92,6 +92,58 @@ describe('DtProgressBar', () => {
     expect(progressComponent.value).toBe(0);
   });
 
+  it('should set progress to 0 if min is greater than or equal to max', () => {
```

Comment:
> should set progress to 0 if min is greater than or equal to max
I would leave out "or equal to" in the title of this test because of the separate test for "equal to"